packages/forms-web-app/src/dynamic-forms/has-appeal-form/journey.js

```diff
@@ -54,48 +54,89 @@ class HasAppealFormJourney extends Journey {
 				.addQuestion(questions.ownsSomeLand)
 				.withCondition(questionHasAnswer(questions.ownsAllLand, 'no'))
 				.addQuestion(questions.ownsRestOfLand)
-				.withCondition(questionHasAnswer(questions.ownsSomeLand, 'yes'))
-				.addQuestion(questions.ownsLandInvolved)
 				.withCondition(
 					questionsHaveAnswers(
 						[
-							[questions.ownsSomeLand, 'no'],
+							[questions.ownsSomeLand, 'yes'],
 							[questions.ownsAllLand, 'no']
 						],
 						{ logicalCombinator: 'and' }
 					)
 				)
-				.addQuestion(questions.identifyingLandowners)
+				.addQuestion(questions.ownsLandInvolved)
 				.withCondition(
 					questionsHaveAnswers(
 						[
-							[questions.ownsRestOfLand, 'some'],
-							[questions.ownsRestOfLand, 'no'],
-							[questions.ownsLandInvolved, 'some'],
-							[questions.ownsLandInvolved, 'no']
+							[questions.ownsSomeLand, 'no'],
+							[questions.ownsAllLand, 'no']
 						],
-						{ logicalCombinator: 'or' }
+						{ logicalCombinator: 'and' }
 					)
 				)
+				.addQuestion(questions.identifyingLandowners)
+				.withCondition(
+					questionHasAnswer(questions.ownsAllLand, 'no') &&
+						((questionHasAnswer(questions.ownsSomeLand, 'yes') &&
+							questionsHaveAnswers(
+								[
+									[questions.ownsRestOfLand, 'some'],
+									[questions.ownsRestOfLand, 'no']
+								],
+								{ logicalCombinator: 'or' }
+							)) ||
+							(questionHasAnswer(questions.ownsSomeLand, 'no') &&
+								questionsHaveAnswers(
+									[
+										[questions.ownsLandInvolved, 'some'],
+										[questions.ownsLandInvolved, 'no']
+									],
+									{ logicalCombinator: 'or' }
+								)))
```

Comment:
this is grueling

Reply:
Not that I lack confidence in this megaternary but it might be nicer to make special functions for working these out with more verbose if statements

Reply:
I think this one is reducible to 
```
		const thingy = () => {
			if (questionHasAnswer(questions.ownsAllLand, 'yes')) return false;
			if (
				questionHasAnswer(questions.ownsSomeLand, 'no') ||
				questionHasAnswer(questions.ownsRestOfLand, 'yes')
			)
				return false;
			if (
				questionHasAnswer(questions.ownsSomeLand, 'yes') ||
				questionHasAnswer(questions.ownsLandInvolved, 'yes')
			)
				return false;
			return true;
		};
```
but I drank a beer while I did it so we should loop back to this on Tuesday

Reply:
much nicer now